plugin/scheduler_example/evict_leader.go

```diff
@@ -273,47 +273,55 @@ func (handler *evictLeaderHandler) UpdateConfig(w http.ResponseWriter, r *http.R
 		args = append(args, handler.config.getRanges(id)...)
 	}
 
-	handler.config.BuildWithArgs(args)
-	err := handler.config.Persist()
+	err := handler.config.BuildWithArgs(args)
 	if err != nil {
-		handler.rd.JSON(w, http.StatusInternalServerError, err.Error())
+		_ = handler.rd.JSON(w, http.StatusBadRequest, err.Error())
+		return
 	}
```

Comment:
How about using `if err := BuildWithArgs(); err != nil` ?

Reply:
Unless all places maintain a habit, either one is fine, no big deal

Reply:
I just noticed that you are using two methods to check for errors😛 
https://github.com/tikv/pd/pull/8301/files#diff-e5d980e123257fd28befe0f6e57d03f2988cacc60a40a22a76afec251f6e53daR313

Reply:
I mean all repo keep one habit~